### PR TITLE
SPARKC-356 fix itegration test classpath

### PR DESCRIPTION
**Pull request by jtgrabowski**
Old guava and slf4j made running integration tests within idea
impossible. This commit removes these two dependencies from test
classpath.

Now it is possible to run/debug integration tests within idea. Before
running tests specify -DbaseDir=<xyz> parameter.

### CODE REVIEW

**Comment by jacek-lewandowski** [comment]
test this please

**Comment by jtgrabowski** [comment]
retest this please

File: project/Versions.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ object Versions {
   val CommonsIO       = "2.4"
   val CommonsLang3    = "3.3.2"
   val Config          = "1.2.1"
-  val Guava           = "16.0.1"
```
Comment (jtgrabowski) [inline project/Versions.scala]: This seems to be delicate matter. Do you have any special corner cases on your mind that I should tests with new guava version?

Comment (RussellSpitzer) [inline project/Versions.scala]: Running with Hadoop? As long as assembly works.
